installation.md

```diff
@@ -34,7 +34,7 @@ Run `hostname -I` on the Raspberry Pi to determine the IP address to use in the
   ```bash
 sudo apt update
 sudo apt upgrade
-sudo apt install firefox-esr xvfb portaudio19-dev libatlas-base-dev redis-server espeak rustc python3-dev
+sudo apt install firefox-esr xvfb portaudio19-dev libatlas-base-dev redis-server espeak rustc python3-dev network-manager
```

Comment:
Leave the installation.md as it was -- we'll update it in a separate pull request